src/utils/index.ts

```diff
@@ -40,6 +38,8 @@ export function getEtherscanLink(
   const prefix =
     networkVersion === PolygonNetworkInfo
       ? 'https://polygonscan.com/'
+      : networkVersion === CeloNetworkInfo
+      ? 'https://explorer.celo.org/'
```

Comment:
im noticing a bug thats surfacing with these links, when i click i go to `https://explorer.celo.org//address/0x085c3685cf4b4ab3d76c28deeb27a2c4e415a0cd` which doesnt render 

I think its actually an issue of how we do the linking when using `getEtherscanLink` (can see this if you click on links to other networks)

So i would try to fix this before launch, OR remove the trailing `/` in celo link if it seems like a big refactor and blocking launch

Reply:
aka i think the best solution is to remove the leading `/` when we construct explorer links with `getEtherscanLink`

Reply:
Just checked, the extra slash exists for polyscan & arbiscan but for some reason doesn't effect it (as it shouldn't)

I'm not sure why the extra slash throws an internal server error for the celo block explorer (reported to my team). 

Removing the extra slash